### PR TITLE
Marklet w/ new connect

### PR DESCRIPTION
**Pull request by dzucconi**
👍

### CODE REVIEW

File: apps/marklet/public/loader.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 (function(){
-
   var markletFrame;
   var markletDiv;
   var markletStyle;
 
+  var markletWidth = '335px'
+  var markletHeightContracted = '460px'
+  var markletHeightExpanded = '710px'
+  var apiEndpoint = 'http://localhost:5000';
```
Comment (dzucconi) [inline apps/marklet/public/loader.js]: Note to self: change this back

File: apps/marklet/routes.coffee [hunk 2]
```diff
@@ -2,20 +2,21 @@
...
+  connections = new Backbone.Collection
+
   res.render 'index',
-    content: content
+    state: state.toJSON()
+    block: block.toJSON()
     isURL: isURL
-    tab: new TabState()
     query: query
+    connections: connections
```
Comment (dzucconi) [inline apps/marklet/routes.coffee]: `toJSON()`

File: apps/marklet/stylesheets/index.styl [hunk 3]
```diff
@@ -1,124 +1,125 @@
...
+//   from
+//     background-position 0 0
+//   to
+//     background-position 160px 80px
+
+// #save--connect .new-connection__button
+//   display flex
+//   justify-content center
+//   .button--inblock
+//     width 100%
```
Comment (dzucconi) [inline apps/marklet/stylesheets/index.styl]: Delete

File: apps/marklet/components/marklet/view.coffee [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    else if src?
+      @model.set 'content', src
+      @state.set 'mode', 'image'
+
+    # Dropped highlighted text
+    else
+      @model.set 'content', data.value['text/plain']
+      @state.set 'mode', 'text'
+
+  saveAndClose: (e) ->
```
Comment (broskoski) [inline apps/marklet/components/marklet/view.coffee]: Nice, this view is so much better.

File: apps/marklet/client/index.coffee [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  model = new Backbone.Model
+    id: uuid()
+    content: CONTENT
+
+  view = new MarkletView
+    el: $('.js-marklet').show()
+    model: model
+
+  view.render()
+  view
```
Comment (broskoski) [inline apps/marklet/client/index.coffee]: Bravo, much nicer.

File: components/connect_v2/client/collections.coffee [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    .then ->
+      search.each (channel) ->
+        recentConnections.append channel
+
+      search.reset recentConnections.toJSON()
+
+  {
+    search: search
+    recentConnections: recentConnections
+  }
```
Comment (broskoski) [inline components/connect_v2/client/collections.coffee]: 😎

**Comment by dzucconi** [comment]
marklet

marklet2

**Comment by broskoski** [comment]
Daaaamn looks 👌

**Comment by dzucconi** [comment]
This should be good to merge (CI is just running on my comment removal)

**Pull request merged**